assets/xml/objects/object_sdn.xml

```diff
@@ -1,66 +1,66 @@
+        <Animation Name="gDemoheishiHalt" Offset="0x57BC" />
+        <Animation Name="gDemoheishiHalt2" Offset="0x5D28" />
+        <Animation Name="gDemoheishiHalt3" Offset="0x64C0" />
+        <Animation Name="gDemoheishiStandHandOnHip" Offset="0x6C18" />
+        <DList Name="gDemoheishiTorsoDL" Offset="0x9EB0" />
+        <DList Name="gDemoheishiHeadDL" Offset="0xA1A8" />
+        <DList Name="gDemoheishiRightShoulderDL" Offset="0xA678" />
+        <DList Name="gDemoheishiRightForearmDL" Offset="0xA768" />
+        <DList Name="gDemoheishiRightHandWithSpearDL" Offset="0xA9A8" />
+        <DList Name="gDemoheishiLeftShoulderDL" Offset="0xADE0" />
+        <DList Name="gDemoheishiLeftForearmDL" Offset="0xAED0" />
+        <DList Name="gDemoheishiLeftHandDL" Offset="0xB120" />
+        <DList Name="gDemoheishiPelvisDL" Offset="0xB438" />
+        <DList Name="gDemoheishiRightThighDL" Offset="0xB5B0" />
+        <DList Name="gDemoheishiRightCalfDL" Offset="0xB698" />
+        <DList Name="gDemoheishiRightFootDL" Offset="0xB988" />
+        <DList Name="gDemoheishiLeftThighDL" Offset="0xBAD0" />
+        <DList Name="gDemoheishiLeftCalfDL" Offset="0xBBB8" />
+        <DList Name="gDemoheishiLeftFootDL" Offset="0xBEB8" />
+        <Texture Name="gDemoheishiSkinTlut" OutName="tlut_00C000" Format="rgba16" Width="16" Height="16" Offset="0xC000" />
+        <Texture Name="gDemoheishiGeneralTlut" OutName="tlut_00C200" Format="rgba16" Width="16" Height="16" Offset="0xC200" />
+        <Texture Name="gDemoheishArmorStrapTex" OutName="tex_00C400" Format="ci8" Width="32" Height="32" Offset="0xC400" />
+        <Texture Name="gDemoheishiArmorPlate1Tex" OutName="tex_00C800" Format="i8" Width="16" Height="16" Offset="0xC800" />
+        <Texture Name="gDemoheishArmorEmblemTex" OutName="tex_00C900" Format="i8" Width="32" Height="32" Offset="0xC900" />
+        <Texture Name="gDemoheishSkinShadowGradient2Tex" OutName="tex_00CD00" Format="ci8" Width="8" Height="8" Offset="0xCD00" />
+        <Texture Name="gDemoheishiShortsToLegTex" OutName="tex_00CD40" Format="ci8" Width="16" Height="16" Offset="0xCD40" />
+        <Texture Name="gDemoheishArmorTex" OutName="tex_00CE40" Format="ci8" Width="8" Height="8" Offset="0xCE40" />
+        <Texture Name="gDemoheishSkinShadowGradient1Tex" OutName="tex_00CE80" Format="ci8" Width="8" Height="8" Offset="0xCE80" />
+        <Texture Name="gDemoheishiLipsTex" OutName="tex_00CEC0" Format="ci8" Width="16" Height="16" Offset="0xCEC0" />
+        <Texture Name="gDemoheishArmorCoatOfArmsTex" OutName="tex_00CFC0" Format="ci8" Width="32" Height="32" Offset="0xCFC0" />
+        <Texture Name="gDemoheishiArmorPlate2Tex" OutName="tex_00D3C0" Format="i8" Width="16" Height="8" Offset="0xD3C0" />
+        <Texture Name="gDemoheishiFingersTex" OutName="tex_00D440" Format="ci8" Width="16" Height="16" Offset="0xD440" />
+        <Limb Name="gDemoUnknownLeftStomachLimb" Type="Standard" Offset="0xD540" />
```

Comment:
```suggestion
        <Limb Name="gDemoheishiRootLimb" Type="Standard" Offset="0xD540" />
```
Looking at `z64Utils` this looks to just be the root limb of the skeleton (all other limbs are its children and it has no DList)

Reply:
Yeah, I was wondering about this. Thanks, I updated it